docs/preview/connected-registry/quickstart-deploy-connected-registry-nested-iot-edge-cli.md

```diff
@@ -0,0 +1,273 @@
+        "$edgeAgent": {
+            "properties.desired": {
+                "modules": {
+                    "connected-registry": {
+                        "settings": {
+                            "image": "10.16.7.4/acr/connected-registry:0.2.0",
+                            "createOptions": "{\"HostConfig\":{\"Binds\":[\"/home/azureuser/connected-registry:/var/acr/data\",,\"/usr/local/share/ca-certificates:/usr/local/share/ca-certificates\",\"/etc/ssl/certs:/etc/ssl/certs\"]}}"
+                        },
+                        "type": "docker",
+                        "env": {
+                            "ACR_REGISTRY_CONNECTION_STRING": {
+                                "value": "ConnectedRegistryName=myconnectedmirror;SyncTokenName=myconnectedmirror-sync-token;SyncTokenPassword=s0meCoMPL3xP4$$W0rd001!@#;ParentGatewayEndpoint=10.16.7.4;ParentEndpointProtocol=https"
+                            }
+                        },
+                        "status": "running",
+                        "restartPolicy": "always",
+                        "version": "1.0"
+                    },
+                    "IoTEdgeApiProxy": {
+                        "settings": {
+                            "image": "10.16.7.4/azureiotedge-api-proxy:latest",
+                            "createOptions": "{\"HostConfig\": {\"PortBindings\": {\"443/tcp\": [{\"HostPort\": \"443\"}]}}}"
+                        },
+                        "type": "docker",
+                        "version": "1.0",
+                        "env": {
+                            "NGINX_DEFAULT_PORT": {
+                                "value": "443"
+                            },
+                            "CONNECTED_ACR_ROUTE_ADDRESS": {
+                                "value": "connectedRegistry:8080"
+                            },
+                            "NGINX_CONFIG_ENV_VAR_LIST": {
+                                    "value": "NGINX_DEFAULT_PORT,BLOB_UPLOAD_ROUTE_ADDRESS,CONNECTED_ACR_ROUTE_ADDRESS,IOTEDGE_PARENTHOSTNAME,DOCKER_REQUEST_ROUTE_ADDRESS"
+                            },
+                            "DOCKER_REQUEST_ROUTE_ADDRESS": {
+                                "value": "registry:5000"
+                            },
+                            "BLOB_UPLOAD_ROUTE_ADDRESS": {
+                                "value": "AzureBlobStorageonIoTEdge:11002"
+                            }
+                        },
+                        "status": "running",
+                        "restartPolicy": "always",
+                        "startupOrder": 3
+                    }
+                },
+                "runtime": {
+                    "settings": {
+                        "minDockerVersion": "v1.25",
+                        "registryCredentials": {
+                            "tsmregistry": {
+                                "address": "10.16.7.4",
+                                "password": "$$$0meCoMPL3xP4$$W0rd001!@#$$",
```

Comment:
It should be the sync token username and password of the nested connected registry.

Reply:
Fixed